src/config.ts

```diff
@@ -0,0 +1,19 @@
+import * as t from "io-ts";
+
+const IO_IM_DEFAULT_LOGGER_LEVEL = "info";
+
+const LogLevel = t.union([
+  t.literal("error"),
+  t.literal("warn"),
+  t.literal("info"),
+  t.literal("http"),
+  t.literal("verbose"),
+  t.literal("debug"),
+  t.literal("silly"),
+]);
+
+type LogLevel = t.TypeOf<typeof LogLevel>;
+
+export const logLevel = LogLevel.decode(
+  process.env.IO_IM_LOGGER_LEVEL,
```

Comment:
pls also add this env to the README for reference, like we did [here](https://github.com/teamdigitale/io-pagopa-proxy#environment-variables)